lib/static-rails/csrf_middleware.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "gets_csrf_token"
+
+module StaticRails
+  class CsrfMiddleware
+    def initialize(app)
+      @app = app
+      @gets_csrf_token = GetsCsrfToken.new
+    end
+
+    def call(env)
+      if env["__static_rails_evil_request_for_csrf_token"]
+        req = Rack::Request.new(env)
+        [200, {}, [@gets_csrf_token.call(req)]].tap do
+        end
+      else
+        @app.call(env)
+      end
+    end
+  end
+end
```

Comment:
Is this file still used anywhere? Seems like it might've been superceded by `SitePlusCsrfMiddleware`?

Loving seeing your experimentation on this, btw! Hope you don't mind the unsolicited comment, I've just been following along and this PR in particular was something I've been looking forward to reading :)

Reply:
fixed!